server/uptime-kuma-server.js

```diff
@@ -197,32 +197,70 @@ class UptimeKumaServer {
     /**
      * Send list of monitors to client
      * @param {Socket} socket Socket to send list on
+     * @param {string} op list, add, update, delete
+     * @param {number} monitorID update or deleted monitor id
      * @returns {Promise<object>} List of monitors
      */
-    async sendMonitorList(socket) {
-        let list = await this.getMonitorJSONList(socket.userID);
-        this.io.to(socket.userID).emit("monitorList", list);
-        return list;
+    async sendMonitorList(socket, op = OPERATIONS.LIST, monitorID = null) {
+        let result = {};
+        let list = {};
+
+        if (op !== OPERATIONS.DELETE) {
+            list = await this.getMonitorJSONList(socket.userID, monitorID);
+        }
+
+        result["op"] = op;
+        result["monitorID"] = monitorID;
+        result["list"] = list;
+        this.io.to(socket.userID).emit("monitorList", result);
+        return result;
     }
 
     /**
      * Get a list of monitors for the given user.
      * @param {string} userID - The ID of the user to get monitors for.
+     * @param {number} monitorID - The ID of monitor for.
      * @returns {Promise<object>} A promise that resolves to an object with monitor IDs as keys and monitor objects as values.
      *
      * Generated by Trelent
      */
-    async getMonitorJSONList(userID) {
+    async getMonitorJSONList(userID, monitorID = null) {
         let result = {};
 
-        let monitorList = await R.find("monitor", " user_id = ? ORDER BY weight DESC, name", [
-            userID,
-        ]);
+        // Initialize query and parameters
```

Comment:
Please make sure that there are comments that explain uncommon/nonobvious logic.
There are a few other instances if this ^^

Here is a longer form expaination[^1]

[^1]: https://www.youtube.com/watch?v=Bf7vDBBOBUA

Reply:
@CommanderStorm 
thank you i will remove unnecessary comment

Reply:
@CommanderStorm 
removed unnecessary commets